odl_video/settings.py

```diff
@@ -436,6 +436,7 @@
 # Celery
 # http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html
 REDIS_URL = get_string("REDIS_URL", None)
+REDIS_MAX_CONNECTIONS = get_int("REDIS_MAX_CONNECTIONS", 2147483648)
```

Comment:
Should we add a more sane default?

Reply:
Changed to 65K (2147483648 is the number I got back on CI, without any specific setting applied).